girder/web_client/test/testEnv.pug

```diff
@@ -8,7 +8,6 @@ html(lang='en')
 
   body
     #g-global-info-apiroot.hide %HOST%#{apiRoot}
-    #g-global-info-staticroot.hide %HOST%#{staticRoot}
```

Comment:
This may be the reason all the web client tests are failing. We could just hard code for the test environment.

Reply:
This `<div>` should now be meaningless. The value of `/static` just needs to be provided to Webpack, which it ostensibly is.

I haven't had a chance to figure out why the tests actually are failing though.